docker/Dockerfile

```diff
@@ -53,6 +55,18 @@ RUN set -e; \
 
 RUN set -e; \
     export DEB_MAX_ATTEMPT=10; \
+    export CRIS_IMG_ARCH="$(set -e; \
```

Comment:
Do you really need to export this?

Reply:
(Doesn't matter, just asking)

Reply:
No, but the format looks better, and we don't need to change it if we add subcommands in the future.

Reply:
Actually I'm asking because I saw those BAZEL_INSTALL_DIR with export.
But still, either way is OK.